src/view/configuration/configuration.jsx

```diff
@@ -441,11 +434,11 @@ const Configuration = ({
     };
   }, [firstInstance.edgeConfigId, firstInstance.edgeConfigInputMethod]);
 
-  useNewlyValidatedFormSubmission(() => {
+  useNewlyValidatedFormSubmission(errors => {
     // If the user just tried to save the configuration and there's
     // a validation error, make sure the first accordion item containing
     // an error is shown.
-    if (isSubmitting && !isValidating && errors && errors.instances) {
+    if (errors && errors.instances) {
```

Comment:
I improved `useNewlyValidatedFormSubmission` so that some of this logic became unnecessary.